### PR TITLE
Code legibility improvement

### PR DESCRIPTION
**Pull request by rodrijuarez**
Moved the declaration from the variable subscribe above the declaration
of the var unsubscribe, this is related to the issue #763

### CODE REVIEW

File: packages/react-server/core/components/RootElement.js [hunk 1]
```diff
@@ -132,28 +132,27 @@ RootElement.ensureRootElement = function(element){
...
-				childProps,
-				subscribe,
-				unsubscribe,
-			}));
-		}
-	});
-	var subscribe = callback => {updater = callback};
+	var dfd = Q.defer(),
+		updater,
+		subscribe = callback => {updater = callback},
```
Comment (gigabo) [inline packages/react-server/core/components/RootElement.js]: Oh, nice.  I _think_ this addresses #763.

**Comment by doug-wade** [comment]
:+1:

**Comment by rodrijuarez** [comment]
Can we close this one? @gigabo

**Comment by gigabo** [comment]
Yep!  Thanks @rodrijuarez!

**Pull request merged**